extension/chrome/elements/compose-modules/formatters/plain-mail-msg-formatter.ts

```diff
@@ -16,7 +16,8 @@ export class PlainMsgMailFormatter extends BaseMailFormatter {
     if (this.richtext) {
       body['text/html'] = newMsg.plainhtml;
     }
-    return await SendableMsg.create(this.acctEmail, { ...this.headers(newMsg), body, atts });
+    //return await SendableMsg.create(this.acctEmail, { ...this.headers(newMsg), body, atts });
+    return await SendableMsg.createPgpMime(this.acctEmail, this.headers(newMsg), body, atts);
```

Comment:
This is wrong. We are sending a plain, unencrypted message. There is no encryption involved. What you'll need is a method like `SendableMsg.createPlain`. Else the signature is the same as this one.

Reply:
Ok